### PR TITLE
Update eric moerth information

### PR DESCRIPTION
**Pull request by EricMoerthVis**
Updating my information on the website and adding myself to the projects I am actively working on.

### CODE REVIEW

File: _members/eric-moerth.md [hunk 1]
```diff
@@ -16,10 +16,6 @@ services:
...
-During his PhD study, Eric Moerth conducted research in multimodal medical visualization.
-His main focus was the research of new and innovative ways to visualize and explore medical data,
-e.g. MRI data to enable doctors to have a better view at their data.
-His projects resulted in successful publications in the field of medical visualization.
+Eric Moerth is a Post Doctoral Specialist Research Fellow in the Department of Biomedical Informatics at Harvard Medical School (DBMI) and part of the HIDIVE Lab. He received his PhD from the University of Bergen in Norway During his PhD study, Eric Moerth conducted research in multimodal medical visualization. His focus was the research of new and innovative ways to visualize and explore medical data. Now Eric is researching the visualization of large-scale 3D Tissue Images using novel rendering techniques. Furthermore, his research enables the efficient analysis of highly multiplexed tissue images, and he also investigates how novel output devices like virtual and augmented reality devices can help investigators make better sense of their data.
```
Comment (lchoy) [inline _members/eric-moerth.md:21]:
```suggestion
Eric Moerth is a Post Doctoral Specialist Research Fellow in the Department of Biomedical Informatics at Harvard Medical School (DBMI) and part of the HIDIVE Lab. He received his PhD from the University of Bergen in Norway. During his PhD study, Eric Moerth conducted research in multimodal medical visualization. His focus was the research of new and innovative ways to visualize and explore medical data. Now Eric is researching the visualization of large-scale 3D Tissue Images using novel rendering techniques. Furthermore, his research enables the efficient analysis of highly multiplexed tissue images, and he also investigates how novel output devices like virtual and augmented reality devices can help investigators make better sense of their data.
```

Minor grammatical fix to add period after second sentence.

File: _members/eric-moerth.md [hunk 2]
```diff
@@ -16,10 +16,6 @@ services:
 start: April 2023
 end:
 ---
-[Eric Moerth](https://vis.uib.no/team/eric-morth/) is a Research Fellow in Biomedical Informatics at Harvard Medical School.
+[Eric Moerth] is a Research Fellow in Biomedical Informatics at Harvard Medical School.
```
Comment (lchoy) [inline _members/eric-moerth.md:19]:
```suggestion
Eric Moerth is a Research Fellow in Biomedical Informatics at Harvard Medical School.
```

The square brackets are not needed if there's no link.

**Review by lchoy — CHANGES_REQUESTED** [review]
A couple minor suggestions.

**Comment by EricMoerthVis** [comment]
Thank you for the suggestions, I commited them

**Review by lchoy — APPROVED** [review]

**Pull request merged**